README.md

```diff
@@ -1,16 +1,33 @@
 # Movies browser
 
+![Demonstration](/Demo.gif)
+
 https://wojtekkkwojtek.github.io/movies-browser-react/
 
 ## About the app
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app) and was created by YouCode students: 
-Dorota Borowska, Miłosz Zajączkowski and Wojciech Kurdzielewicz.
+* [Dorota Borowska](https://github.com/IT2021db) 
```

Comment:
It's a good idea to send viewer to GitHub repositories of the project cooperators